gem/lib/pupperware/spec_helper.rb

```diff
@@ -419,6 +433,21 @@ def curl_pe_orchestration_services(end_point)
     curl('localhost', 8143, end_point).body
   end
 
+  def orchestrate_puppet_run(target_agent: 'puppet-agent.test', network: 'pupperware-commercial', rbac_username: 'admin', rbac_password: 'admin', puppetserver: 'puppet.test', pe_console_services: 'pe-console-services.test', pe_orchestration_services: 'pe-orchestration-services.test')
+    output = run_command("docker run \
+           --rm \
+           --network #{network} \
+           --env RBAC_USERNAME=#{rbac_username} \
+           --env RBAC_PASSWORD=#{rbac_password} \
+           --env PUPPETSERVER_HOSTNAME=#{puppetserver} \
+           --env PE_CONSOLE_SERVICES_HOSTNAME=#{pe_console_services} \
+           #{CLIENT_TOOLS_IMAGE} \
```

Comment:
This should be defined.........should we just move it out of the existing spot in the test file and hardcoded into this method?